### ISSUE TITLE
Verständnisfrage zum Modul

### ISSUE THREAD

**Issue by Shoekrates** [Tastaturberuf/contao-doccheck-login-bundle#12]
Hallo!
Habe ich das richtig verstanden, dass über diesen Login ausschließlich ein allgemeiner Login realisiert werden kann, ohne nach dem eingeloggten DocCheck-User zu differenzieren?
Also ich kann in Contao dann keine Mitglieds-individuellen Inhalte aufbauen und per Login schützen, da ein festes Contao-Mitglied im Modul definiert wird?
Zweite Frage: notwendig auf DocCheck-Seite ist nur die Basic-Lizenz, oder? Oder gewinnt man durch ein höheres Lizenz-Modell irgendwas im Zusammenspiel mit dem Modul?

**Comment by Tastaturberuf** [comment]
Hallo Christian,

DocCheck selbst differenziert nicht nachdem welcher User eingeloggt wird, nur ob die "höhe" der Rechte stimmen. Danach wird einfach nur stumpf weitergeleitet. 

Und ja, die Basis-Lizenz reicht aus.

**Comment by Shoekrates** [comment]
Danke für die schnelle Antwort!
Heißt aber auch, dass wir neben dem DocCheck-Login keine individuellen Inhalte auf der Contao-Seite vorhalten können?
Denn ein „Login im Login“ (DocCheck-Login im Loginbereich mit bereits eingeloggtem Contao-Mitglied würde ja nicht funktionieren, weil keine zwei Contao-Mitglieder gleichzeitig eingeloggt sein können, oder? Oder habe ich da einen Denkfehler?

**Comment by Tastaturberuf** [comment]
Laut Doku geht das Weiterleiten jetzt doch, aber ich glaube nicht in der Basis-Version, die war schon krass kastriert. Die Erweiterung kann das aber nicht. 

Quelle: https://more.doccheck.com/fileadmin/user_upload/files/industry/b2b-landingpage/industry-erste-hilfe-kasten-technisches-handbuch.pdf

**Comment by Tastaturberuf** [comment]
> Danke für die schnelle Antwort! Heißt aber auch, dass wir neben dem DocCheck-Login keine individuellen Inhalte auf der Contao-Seite vorhalten können? Denn ein „Login im Login“ (DocCheck-Login im Loginbereich mit bereits eingeloggtem Contao-Mitglied würde ja nicht funktionieren, weil keine zwei Contao-Mitglieder gleichzeitig eingeloggt sein können, oder? Oder habe ich da einen Denkfehler?

Zwei Mitglieder anmelden geht nicht. Das ganze DocCheck-Prinzip ist letztlich auch kaputt, eigentlich meldet man nicht wirklich einen Benutzer an, sondern leitet auf eine "geheime" Seite weiter. 

Wenn du mehrere Zielgruppen hast, kannst du aber mehrere Login-Module mit unterschiedlichen Zielen benutzen.

**Comment by Tastaturberuf** [comment]
Ich denke für einen Usecase ist es am besten keine Erweiterung dafür einzusetzen, sondern einfach die DocCheck-Iframes manuell einzubinden.

**Comment by Shoekrates** [comment]
Hmm. Das wäre auch noch eine Idee. Ich sehe, da ist noch einiges für den Prozess mit dem Kunden zu klären. 
Danke auf jeden Fall für die Infos. Darf ich auf dich zurückkommen, wenn wir Probleme bei der Umsetzung haben?

**Comment by Tastaturberuf** [comment]
Ja, aber erst im Frühjahr.

**Comment by akroii** [comment]
> Hallo! Habe ich das richtig verstanden, dass über diesen Login ausschließlich ein allgemeiner Login realisiert werden kann, ohne nach dem eingeloggten DocCheck-User zu differenzieren? Also ich kann in Contao dann keine Mitglieds-individuellen Inhalte aufbauen und per Login schützen, da ein festes Contao-Mitglied im Modul definiert wird? Zweite Frage: notwendig auf DocCheck-Seite ist nur die Basic-Lizenz, oder? Oder gewinnt man durch ein höheres Lizenz-Modell irgendwas im Zusammenspiel mit dem Modul?

Richtig. Es gibt bei DocCheck je nach Berufsgruppe die Möglichkeit verschiedene "geheime Seiten fertzulegen". Bei unserer Variate kommen alle Nutzer auf ein und die selbe Seite. Das entspricht der Basic-Lizenz.
![image](https://user-images.githubusercontent.com/8830861/208428324-f5b6c3b0-c3b8-482e-84da-123fcc311c6a.png)
Das dürfte auch deine zweite Frage klären. Bisher kann unsere Erweiterung das nicht.

Jedoch müssten wir hier unsere Erweiterung etwas pimpen.
